src/components/live-session-view.tsx

```
@@ -0,0 +1,1173 @@
+                      </div>
+                    )}
+                    <div
+                      className="prose prose-lg max-w-none leading-relaxed text-gray-800"
+                      style={{
+                        lineHeight: "1.8",
+                        fontSize: "16px",
+                      }}
+                    >
+                      <div
+                        dangerouslySetInnerHTML={{
+                          __html: currentContent.content
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)